release_notes.md

```
@@ -0,0 +1,79 @@
+ - PR Link: https://github.com/bluewave-labs/bluewave-onboarding/pull/205
+ - Merged on: 2024-09-14
+ - Linked Issues:
+   - Axios errors return undefined: https://github.com/bluewave-labs/bluewave-onboarding/issues/204
+
+## 102 forget password backend fix
+ - Created by: uparkalau
+ - PR Link: https://github.com/bluewave-labs/bluewave-onboarding/pull/200
+ - Merged on: 2024-09-14
+
+## 191 request loop
+ - Created by: erenfn
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)